### PR TITLE
517 - Adding husky configuration

### LINKED ISSUE #517: Add husky config
Husk is a tool that helps improving code quality by running tasks like testing and linting your code before doing a commit or push

### PR DESCRIPTION
**Pull request by andersonmalheiro**
fix #517

# Adding husky configuration

#### Only select the appropriate.

- [ ] **Model testing.**
- [ ] **Request testing and swagger doc update.**
- [ ] **System testing.**
- [x] **No tests required.**

### CODE REVIEW

File: frontend/.lintstagedrc.json
Comment (iurynogueira) [inline frontend/.lintstagedrc.json:4]: What do you think about using yarn as default?

**Review by iurynogueira — CHANGES_REQUESTED** [review]

**Comment by andersonmalheiro** [comment]
⚠️  Don't merge until #529 its merged

**Comment by andersonmalheiro** [comment]
Closing because I have made a new branch with a cleaner history

**Pull request closed without merging**
